src/main/java/org/janelia/saalfeldlab/n5/N5Writer.java

```diff
@@ -162,4 +162,19 @@ public default void createDataset(
 			final String pathName,
 			final DatasetAttributes datasetAttributes,
 			final DataBlock<T> dataBlock) throws IOException;
+
+	/**
+	 * Deletes the block at {@code gridPosition}
+	 *
+	 * @param pathName dataset path
+	 * @param gridPosition position of block to be deleted
+	 * @throws IOException
+	 *
+	 * TODO what is a good return value?
+	 * @return true if block was deleted by this method; false if the block could not be deleted because it did not exist
```

Comment:
change comment to reflect desired behavior: true if block does not exist (or is 'empty` in backends that do not support sparse data) after the method terminates.  This may include that the block did not exist in the first place.

Reply:
> or is 'empty` in backends that do not support sparse data

Does that mean that the written block should be loaded and checked for zero (or whatever empty means)? I don't think that that would be a good idea. In these cases it is probably best to return `true` in all cases.